src/amo/sagas/users.js

```diff
@@ -62,6 +64,19 @@ export function* editUserAccount({
     });
 
     yield put(loadUserAccount({ user }));
+
+    if (Object.keys(notifications).length) {
```

Comment:
You could also just do `if (notifications !== {})`, I think, although I'm not sure that's any better.

Reply:
mmh for some reasons, I have to think more when reading `notifications !== {}` than `.length`, because we use `.length` for arrays and strings already.